operator/api/v1alpha1/httpscaledobject_types.go

```diff
@@ -115,6 +115,8 @@ type HTTPScaledObjectSpec struct {
 	// (optional) Replica information
 	//+optional
 	Replicas ReplicaStruct `json:"replicas,omitempty"`
+	//(optional) Target metric value
+	TargetMetric int32 `json:"targetMetric,omitempty" description:"The target metric value for the HPA (Default 100)"`
```

Comment:
@ajanth97 you'll need to change this to `TargetPendingRequests` as well, and then re-generate the CRD